logstash-core/api/lib/app.rb

```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
+require "json"
```

Comment:
We should not use the "json" library but instead our `LogStash::Json` which uses jrjackson.